FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -382,33 +400,44 @@ - (void)fetchLatestConfig:(NSInteger)remainingAttempts targetVersion:(NSInteger)
   __weak RCNConfigRealtime *weakSelf = self;
   dispatch_async(_realtimeLockQueue, ^{
     __strong RCNConfigRealtime *strongSelf = weakSelf;
-    [strongSelf->_configFetch
-        fetchConfigWithExpirationDuration:0
-                        completionHandler:^(FIRRemoteConfigFetchStatus status, NSError *error) {
-                          if (status == FIRRemoteConfigFetchStatusSuccess) {
-                            if ([strongSelf->_configFetch.templateVersionNumber integerValue] >=
-                                targetVersion) {
-                              for (RCNConfigUpdateCompletion listener in strongSelf->_listeners) {
-                                listener(nil);
+    if (strongSelf->_settings.isFetchInProgress) {
+      if (strongSelf->_settings.lastFetchStatus == FIRRemoteConfigFetchStatusSuccess) {
```

Comment:
what happens if fetch status is not success or version < target version? for those cases I think we should still attempt to fetch?

Reply:
Updated for this case.